.github/workflows/prerequisite-test.yml

```diff
@@ -0,0 +1,14 @@
+name: Prerequisite Checks
```

Comment:
Revert this change

Reply:
complete by rewriting the changes to this branch (with the changes from main & the edit to the button needed to close this ticket).